Assets/Scripts/Combat/Hitbox.cs

```diff
@@ -2,10 +2,17 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class Hitbox : MonoBehaviour
+//  DisableSpriteRender causes the sprite to be disabled in runtime builds, but not the editor.
+// This is useful for hitboxes, or other objects that you don't want to see in the game, but you do want to see in the editor.
+public class Hitbox : DisableSpriteRender
```

Comment:
Based on the Composition over Inheritance approach, why wouldn't we just add a `DisableSpriteRenderer` to any GO that needs it? This assumes that every hitbox GO will also have a SpriteRenderer. If someone forgets to add the SpriteRenderer then this will error out in the `DisableSpriteRenderer` Start method.